frontend/src/common/constansts/paths.ts

```diff
@@ -13,6 +14,11 @@ export enum UserPaths {
     SIGN_UP_USER = "/signup"
 }
 
+export enum WalletPaths {
+    WALLET = "/wallets",
+    WALLET_BY_USER = "/users"
+}
```

Comment:
Don't mix in one file your frontend paths and backend endpoints. It's confusing a lot